build_raspOVOS.sh

```
@@ -92,6 +92,11 @@ chmod +x /usr/libexec/ovos-i2csound
 
 ln -s /etc/systemd/system/i2csound.service /etc/systemd/system/multi-user.target.wants/i2csound.service
 
+cp -v /mounted-github-repo/tuning/99-usb-audio.rules  /etc/udev/rules.d/99-usb-audio.rules
+cp -v /mounted-github-repo/tuning/update-audio-sinks.sh  /usr/libexec/update-audio-sinks.sh
+cp /mounted-github-repo/tuning/combine_sinks.service/etc/systemd/system/combine_sinks.service
```

Shellcheck (0.10.0)

[error] This cp has no destination. Check the arguments.

(SC2225)